stable/heapster/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-service
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app: {{ template "fullname" . }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  - port: {{ .Values.service.externalPort }}
+    targetPort: {{ .Values.service.internalPort }}
+    protocol: TCP
+    name: {{ .Values.service.name }}
```

Comment:
It would be somewhat helpful to allow the override of the service name through something like 
```
 {{ if .Values.service.nameOverride}}
  name: {{.Values.service.nameOverride}}
  {{- else }}
  name: {{ template "fullname" . }}
  {{- end }}
```
and removing the `name` attribute of this port.  

By doing so, you could deploy this and "just have it work" with the Kuberentes-Dashboard deployment.  As it stands now, you would need to customize the `--heapster-host` on the Kubernetes deploy to match this explicitly.